### PR TITLE
feat: add tts sample for long audio synthesis

### PR DESCRIPTION
**Pull request by jameszhao00**
Add a new sample for texttospeech long audio synthesis.

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: texttospeech/snippets/synthesize_long_audio.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# Copyright 2018 Google LLC
```
Comment (kweinmeister) [inline texttospeech/snippets/synthesize_long_audio.py:2]: Use new copyright date 2023 for new samples

File: texttospeech/snippets/synthesize_long_audio.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+# [START tts_synthesize_long_audio]
+def synthesize_long_audio(parent, text, output_gcs_uri):
+    """Synthesizes speech from the input string of text."""
+    from google.cloud import texttospeech
+    
+    client = texttospeech.TextToSpeechLongAudioSynthesizeClient()
+
+    input_text = texttospeech.SynthesisInput(text=text)
+
+    voice = texttospeech.VoiceSelectionParams(
```
Comment (kweinmeister) [inline texttospeech/snippets/synthesize_long_audio.py:36]: Consider making these constants, at the top of the sample. (Same comment applies to other options specified throughout sample.)

File: texttospeech/snippets/synthesize_long_audio_test.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+# All Rights Reserved.
+
+import os
+
+import synthesize_long_audio
+
+def test_synthesize_text(capsys):
+    response = synthesize_long_audio.synthesize_long_audio("projects/p0/locations/global", 
+                                                           "hello",
+                                                           "gs://bucket/output.wav")
```
Comment (kweinmeister) [inline texttospeech/snippets/synthesize_long_audio_test.py:25]: We should access sample data from the same location as other TTS samples (a standard Cloud sample data bucket)

(Also same comments about copyright year + constants)

File: texttospeech/snippets/synthesize_long_audio_test.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# All Rights Reserved.
+
+import os
+
+import synthesize_long_audio
+
+def test_synthesize_text(capsys):
```
Comment (kweinmeister) [inline texttospeech/snippets/synthesize_long_audio_test.py:22]: The sample method should return a value. And your test should check that the value is expected.

File: texttospeech/snippets/synthesize_long_audio_test.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    response = synthesize_long_audio.synthesize_long_audio("projects/p0/locations/global", 
+                                                           "hello",
+                                                           "gs://bucket/output.wav")
+
```
Comment (engelke) [inline texttospeech/snippets/synthesize_long_audio_test.py:26]: There is no such bucket available. Also, there are no tests here.

**Review by engelke — CHANGES_REQUESTED** [review]
Work out where to store and fetch the sample data, and add at least one test (assert).

**Pull request closed without merging**